the matrials in this folder belong to the course below and can be downloaded from:

http://www.bioconductor.org/help/course-materials/


About the Course

The one-week intensive course Statistical Data Analysis for Genome Biology teaches statistical and computational analysis of multi-omics studies in biology and biomedicine. It covers the underlying theory and state of the art (the morning lectures), and practical hands-on exercises based on the R / Bioconductor environment (the afternoon labs). The course covers the primary analysis of high-throughput sequencing based assays in functional genomics and integrative methods including efficiently operating with genomic intervals, statistical testing, linear models, machine learning, bioinformatic annotation and visualization. At the end of the course, you should be able to run analysis workflows on your own (multi-)omic data, adapt and combine different tools, and make informed and scientifically sound choices about analysis strategies.

The course is intended for researchers who have basic familiarity with the experimental technologies and their applications in biology, and who are interested in making the step from a user of bioinformatics software towards adapting or developing their own analysis workflows. The four practical sessions of the course will require you to follow and modify scripts in the computer language R. To freshen up your R, see below for some links.
Topics

    Introduction to Bioconductor
    Elements of statistics: hypothesis testing, multiple testing, regression, regularization, clustering and classification, parallelization and performance (machine learning), visualisation
    Reproducible research and R authoring with markdown and knitr
    RNA-Seq data analysis and differential expression
    New workflows for RNA-seq
    Computing with sequences and genomic intervals
    End-to-end RNA-Seq workflow
    Experimental design, batch effects and confounding
    Working with annotation – genes, genomic features and variants
    Visualization, the grammar of graphics and ggplot2
    Use of Git and GitHub with R, RStudio, and R Markdown
    Gene set enrichment analysis

Course Structure

The course consists of

    Morning lectures: 20 x 45 minutes: Monday to Friday 8:30 – 12:00
    Practical computer tutorials in the afternoons (14:00 – 17:00) on Monday, Tuesday, Thursday and Friday

Course Materials

The course material will be provided in electronic form during the course. To prepare, please consult material from the 2015 course, material from other related courses in 2015 and 2014, the general Bioconductor documentation and specific package vignettes.
Computer Tutorials

You will work on the labs at your own pace in small groups with expert guidance (all lecturers from the morning sessions plus teaching assistants).

Participants are required to bring their own laptop with the most recent release versions of R (i.e. 3.3) and Bioconductor (i.e. 3.3) installed. If its hardware is sufficiently powered for ‘real work’, you will enjoy the course more. A WiFi network card is welcome to connect to our local network, however, no connections to the internet will be possible from the WLAN at the course venue.
Social Program

Wednesday afternoon is for a joint cultural and outdoors activity. We plan a guided tour at one of the Brixen area’s impressive cultural / historical sites, followed by a trip into the mountains, a (light) walk in the high-alpine area and delicious local dinner.
No Internet Access

Please note: the course venue does not provide stable internet access. The course provides a local network with the course material.
R Basics

    R-Intro from the R Project
    R-Studio, a very useful integrated development environment (IDE)
    Contributed documentation on on CRAN, notably:
        Applied Statistics for Bioinformatics Using R
        IcebreakeR
        A (very) short Introduction to R
    Best first R tutorial
    Teaching material from Laurent Gatto

